test/qt3tests.js

```diff
@@ -30,12 +30,12 @@ if (!(typeof window === 'undefined')) {
 	}
 	else {
 		shouldRunTestByName = require('text-loader!./runnableTestSets.csv')
-			.split('\n')
+			.split('/\r?\n/')
```

Comment:
@ymor Change looks robust to me. Did you consider using https://nodejs.org/api/os.html#os_os_eol instead?

Reply:
Can't say I've really used node before! But that does look like the better solution.

Reply:
I'm no expert either. @bwrrp, @ThomasGravekamp, @DrRataplan recommendations?

Reply:
Given the mess that end-of-line handling can be (depending on git settings, we could get \n on Windows), I guess it's probably best to keep the regex and accept both types here.

Reply:
I have noticed that I didn't add the regex to the "on node" side of the if/else.  Want me to update that too?

Reply:
Good catch! I guess those would also need to be changed for the tests to run successfully on Windows.

Reply:
I'd assume so but I'm not sure how to run the tests purely in node.  Btw that change to the regex you did, means i can actually fully run the qt3 tests locally on windows now. 👍

Reply:
Good to hear. I think the node-specific code is not currently active (we'd eventually like to run all tests in node as well as the browser). I've updated the splits anyway, to hopefully prevent the issue returning when we enable this.